_includes/gift-card-authentication.md

```diff
@@ -0,0 +1,123 @@
+## Authentication
```

Comment:
Instead of prefixing files with `gift-card-`, you can put them in a sub-directory called `gift-card`. Nothing that stops this PR, just a tip.